src/services/identity/DeploymentsService.ts

```diff
@@ -115,6 +118,83 @@ class DeploymentsService {
           .map(() => amplifyInfo)
       })
   }
+
+  getDeploymentInfoFromSiteId = async (repoId: string) => {
+    const deploymentInfo = await this.repository.findOne({
+      where: {
+        siteId: repoId,
+      },
+    })
+    if (!deploymentInfo)
+      return errAsync(new NotFoundError("Site has not been deployed!"))
+    return okAsync(deploymentInfo)
+  }
+
+  updateAmplifyPassword = async (
+    repoName: string,
+    encryptedPassword: string,
+    iv: string,
+    enablePassword: boolean
+  ) => {
+    const deploymentInfo = await this.repository.findOne({
```

Comment:
hey, this can be made simpler right?
eg:
```
    const deploymentInfo = await this.repository.findOne({
      where: {
        name: repoName,
      },
    })
```

Reply:
repoName only exists in `Repo` though, so we need to do the joins!

Reply:
https://github.com/isomerpages/isomercms-backend/pull/806/commits/1703033f5753b37cbeb243c9fcf05557df4474b4